src/webview_provider/docsEditPanel.ts

```diff
@@ -383,9 +384,26 @@ export class DocsEditViewPanel implements WebviewViewProvider {
     if (!data.length) {
       return;
     }
-    return {
-      tests: data,
-    };
+    const dbtVersion = project.getDBTVersion();
+    if (dbtVersion && dbtVersion[0] >= 1 && dbtVersion[1] >= 8) {
+      if (
+        existingColumn &&
+        existingColumn.name === columnNameFromWebview &&
+        existingColumn?.tests === undefined
+      ) {
+        return {
+          data_tests: data,
+        };
+      } else {
```

Comment:
this else is not needed. it will be covered by bottom else

Reply:
we will need it in case it is greater than 1.8 but already has the `tests` field